src/ruwaterloo_clj_scraper/core.clj

```diff
@@ -226,6 +227,12 @@
    ])
 
 
+(defn exit
+  [exit-message exit-code]
+  (do
+    (println exit-message)
+    (System/exit exit-code)))
+
 ;; inspired by https://github.com/clojure/tools.cli
 (defn validate-args
```

Comment:
Rename this to `validate-args!` because it has side effects. The `!` is a naming convention for functions that have side effects, like mutation or exiting the program.

Reply:
Done, thanks for the info.